ansible-advancedlinux/azuredeploy.json

```diff
@@ -226,31 +232,13 @@
     },
     "currentEnvironmentSettings": "[variables('ansibleVmTypes')[parameters('storageAccountType')]]",
     "ansibleVmType": "[variables('currentEnvironmentSettings').vmSize]",
-    "publicIPRef": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPName'))]",
-    "NICipconfig": "ipCnfgBE",
-    "NICRef": "[resourceId('Microsoft.Network/networkInterfaces',variables('vmNICNamePattern'))]",
-    "DNSNameLB": "[concat(parameters('dnsNameLabel'))]",
-    "vnetRef": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
-    "subnetBackEndRef": "[concat(variables('vnetRef'),'/subnets/',variables('subnet2Name'))]",
-    "FrontEndRef": "[concat(variables('vnetRef'),'/subnets/',variables('subnet1Name'))]",
-    "loadBalancerRef": "[resourceId('Microsoft.Network/loadBalancers',variables('loadBalancerName'))]",
-    "lbRuleName": "lbRuleCbAdmin",
-    "lbRuleRef": "[concat(variables('loadBalancerRef'),'/loadBalancingRules/',variables('lbRuleName'))]",
-    "lbFEConfig": "PLBcbaFrontConfg",
-    "lbFEConfigRef": "[concat(variables('loadBalancerRef'),'/frontendIPConfigurations/',variables('lbFEConfig'))]",
-    "lbBEConfig": "[concat('LBBEcbaFrontConfg')]",
-    "lbBEConfigRef": "[concat(variables('loadBalancerRef'),'/backendAddressPools/' ,variables('lbBEConfig'))]",
-    "lbcbaProbe": "cbaLBProbe",
-    "lbcbaProbeRef": "[concat(variables('loadBalancerRef'),'/probes/',variables('lbcbaProbe'))]",
-    "lbProbePort": 8092,
-    "lbExposedPort": 80,
-    "lbcbaIdleTimeOut": 4,
-    "newStorageAccountName": "[concat(resourceGroup().name,parameters('newStorageAccountName'),'0')]",
+    "newStorageAccountName": "[concat(parameters('dnsNameLabel'),'stg')]",
     "storageRef": "[concat('Microsoft.Storage/storageAccounts/', variables('newStorageAccountName'))]"
   },
+
   "resources": [
     {
-      "apiVersion": "[variables('apiVersion')]",
+      "apiVersion": "2015-05-01-preview",
```

Comment:
Any reason for going back to hardcoding the apiVersion? Also please use the GA API version 2015-06-15